src/test/java/com/google/api/codegen/testdata/csharp/csharp_library.baseline

```diff
@@ -12763,11 +12763,11 @@ namespace Google.Example.Library.V1
     public partial class Book
     {
         /// <summary>
-        /// <see cref="BookName"/>-typed view over the <see cref="Name"/> resource name property.
+        /// <see cref="Google.Example.Library.V1.BookName"/>-typed view over the <see cref="Name"/> resource name property.
         /// </summary>
-        public BookName BookName
+        public Google.Example.Library.V1.BookName BookName
```

Comment:
This part doesn't need to be fully-qualified. It can only be a type name, so that's how it's looked up.
It doesn't do any harm though.

Reply:
Yes, some of these fully-qualified names could be simplified; however that would entail a much larger change, so I'll keep it like this for now.